### PR TITLE
Simplify basic logics

### PR DESCRIPTION
**Pull request by ebraminio**

### CODE REVIEW

File: app/src/main/java/ir/kindnesswall/utils/extentions/String.kt [hunk 1]
```diff
@@ -49,15 +49,7 @@ fun String.isValidVerificationCode(): Boolean {
...
-fun String.isNumeric(): Boolean {
-    var i = length
-    while (--i >= 0) {
-        if (!Character.isDigit(this[i])) {
-            return false
-        }
-    }
-    return true
-}
+fun String.isNumeric(): Boolean = this.toCharArray().all(Character::isDigit)
```
Comment (beigirad) [inline app/src/main/java/ir/kindnesswall/utils/extentions/String.kt]: You can use `CharSequence.all { it.isDigit() }`

File: app/src/main/java/ir/kindnesswall/utils/extentions/String.kt [hunk 2]
```diff
@@ -141,12 +133,7 @@ fun String.boldAndColorStringHtml(string:String=this,color: Int = Color.BLACK):S
     return formattedText
 }
 
-fun String.isUniCode(): Boolean {
-    for (c in this.toCharArray()) {
-        if (c.toInt() > 255) return true
-    }
-    return false
-}
+fun String.isUnicode(): Boolean = this.toCharArray().any { it.code > 255 }
```
Comment (beigirad) [inline app/src/main/java/ir/kindnesswall/utils/extentions/String.kt]: same

File: app/src/main/java/ir/kindnesswall/utils/extentions/String.kt (hunk 1 above)
Comment (ebraminio) [reply app/src/main/java/ir/kindnesswall/utils/extentions/String.kt]: Done

File: app/src/main/java/ir/kindnesswall/utils/extentions/String.kt (hunk 2 above)
Comment (ebraminio) [reply app/src/main/java/ir/kindnesswall/utils/extentions/String.kt]: Done

**Pull request merged**